verification/ci-verify.py

```
@@ -0,0 +1,1555 @@
+# ---------------------------------------------------------------------------
+
+def _find_first_output_file(outdir: str) -> str | None:
+    """Return the path of the first regular file found in outdir, or None."""
+    for name in sorted(os.listdir(outdir)):
+        path = os.path.join(outdir, name)
+        if os.path.isfile(path):
+            return path
+    return None
+
+
+def run_comparison(
```

Ruff (0.15.17)

[warning] Too many return statements (8 > 6)

(PLR0911)